core/src/main/resources/hudson/model/DirectoryBrowserSupport/dir.jelly

```diff
@@ -71,6 +71,9 @@ THE SOFTWARE.
                   </td>
                   <j:if test="${!x.folder}">
                     <td class="fileSize">
+                      ${h.rfc822Date(x.getLastModifiedAsCalendar())}
```

Comment:
Should just be made into a Groovy view rather than add view-oriented methods into `Path`

Reply:
I will use the same way used in https://github.com/jenkinsci/jenkins/blob/08def67a18eee51de9f3f99bc2a792fee1c160e0/core/src/main/resources/hudson/widgets/HistoryWidget/entry.jelly#L44